ste/JobPartPlan.go

```diff
@@ -389,6 +390,10 @@ type JobPartPlanTransfer struct {
 	// atomicErrorCode has a default value (0) which means either there was no error or transfer failed because some non storageError.
 	// atomicErrorCode should not be directly accessed anywhere except by transferStatus and setTransferStatus
 	atomicErrorCode int32
+
+	// errorMessageLength represents the length of the error message for failed transfers.
+	errorMessageLength int32
+	errorMessage       [MaxErrorMessageLength]byte
```

Comment:
Please correct me if I am wrong:
From what I understand by reading the code, every transfer will have 1000 characters reserved for storing error message. So, we're looking at (1000B * Number of file trasfers in a job) of extra storage. For 1Million file transfers, the storage can be 1GB

Reply:
`func (jpfn JobPartPlanFileName) Create(order common.CopyJobPartOrderRequest)` - this is the function which creates the plan file and I don't see any entry for error code and error message there.